pkgs/_test_macros/lib/built_value.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
Please remove this unused import also :) thanks.

Reply:
sorry :weary: